src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random number = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private final int lowerBound = 1;

```

Comment:
You should make **all** magic numbers **CONSTANTS**
```suggestion
    private static final int LOWER_BOUND = 1;

```